src/components/CuratorDashboard/index.tsx

```
@@ -0,0 +1,125 @@
+			}
+		} else {
+			setOpenLoginPrompt(true);
+		}
+	};
+
+	return (
+		<div className={className}>
+			<main className='mx-3 mt-3'>
+				<div className='flex items-center justify-between'>
+					<span className={`text-3xl ${spaceGrotesk.className} ${spaceGrotesk.variable} font-bold text-bodyBlue dark:text-blue-dark-high`}>Curator Dashboard</span>
+					{!isNaN(id || 0) && !!loginAddress?.length && (
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)